docs/source/guide/ddd-5-theory.myst

```diff
@@ -4,12 +4,91 @@ jupytext:
     extension: .myst
     format_name: myst
     format_version: 0.13
-    jupytext_version: 1.11.4
+    jupytext_version: 1.11.1
 kernelspec:
   display_name: Python 3
   language: python
   name: python3
 ---
 
-
 # What is the theory behind DDD?
+
+Dynamical decoupling (DD) {cite}`Viola_1998_PRA, Viola_1999_PRL, Zhang_2014_PRL`
+is a quantum control technique to effectively reduce the interaction of a quantum system with its environment.
+The protocol works by driving a quantum system with rapid sequences of periodic control pulses.
+The application of DD sequences can have in two effects depending on the correlation time of the environment:
+
+1. For Markovian noise, DD can make the overall quantum channel more symmetric (analogously to quantum twirling)
+but cannot actually decouple the system from the environment;
+
+2. For non-Markovian noise, DD can effectively decouple the system form the environment.
+In theory, for ideal sequences of infinitely quick and strong pulses, complete noise suppresion is achievable.
+
+In real practical settings, due to the finite frequency and finite amplitude of DD sequences,
+both effects are possible but only as imperfect approximations.
+
+In the context of quantum computing, DD can be considered as an error mitigation method.
+With respect to other error mitigation techniques, DDD has very peculiar features:
```

Comment:
Haven't yet introduced the difference between DD and DDD. Might be confusing? Might be fine.

Reply:
DDD should actually be DD in that sentence. Good catch! Changed here [9578789](https://github.com/unitaryfund/mitiq/pull/1311/commits/95787896e97791b4bc5ec369f2343db29ebdc55e)